### DISCUSSION TITLE
Skyrim

### DISCUSSION THREAD

**Discussion by Heliumrich** [skyline-emu/skyline#451] [Game Requests]
Because why not ?

While we wait for an official port to the 47th platform skyrim will be available to

Thanks <3

**Comment by xXxNobuoxXx** [comment]
Skyrim would be great on android! Would be awesome to play it on my Galaxy S10+

**Reply by SUPAMAVERIK** [reply]
It might need more than that.

**Reply by xXxNobuoxXx** [reply]
Like more than a Galaxy S10+? Maybe.. I do run Sonic fairly well and I know it's a lower end game to run. I'm hoping though to play it on mine for now until I can upgrade.. Either Skyrim or BoTW would be great or any other game lol.

**Reply by Swizzlefowl** [reply]
I don't think galaxy s10+ would cut it for Skyrim or botw you will have to get a better phone then that.

**Reply by xXxNobuoxXx** [reply]
Well.. one can hope right.. 😀

**Comment by 420MaryJane** [comment]
I'm literally ready to wait 10 years for Skyrim to be playable on Skyline, just as long as I know you guys are working on it.🔥

**Comment by raviii12** [comment]
Dont worry guys within 1yr we can play almost 70% switch games on Android with this emulator like aethsx2 emulator.

**Comment by Raxcental** [comment]
I agree with this. Playing Skyrim on my Duo 2 would be amazing. Skyrim via Skyline on one screen and then youtube or something else on the other :). Please tell me you fine folks are working on it.

**Comment by raviii12** [comment]
Iam just waiting for www18

**Reply by Raxcental** [reply]
www18?

**Comment by raviii12** [comment]
Wwe18 dude
